BUILD.bazel

```diff
@@ -23,3 +23,18 @@ filegroup(
     # reference our installed node_modules directory.
     visibility = ["//visibility:public"],
 )
+
+load("@angular//:index.bzl", "ng_module", "ng_package")
+
+ng_module(
+    name = "root_module",
+    srcs = glob(["*.ts"]),
+    deps = ["//src"],
+)
+
+ng_package(
```

Comment:
Does the `ng_package` rule also allow to configure the output folder?

Reply:
what kind of configuration do you mean?

Reply:
Like to define a relative path `./dist` or something where the output will be placed. Right now on my workstation it is somewhere here `/private/var/tmp/_bazel_juri/ff20d34e5c6e607a9b98045b3a379a98/execroot/ngx_tabs_libdemo/bazel-out/darwin-fastbuild/bin/ngx-tabs-lib` 😄

Reply:
Bazel always places outputs in the bazel-out folder, after the build is finished you can copy them.
However, you may as well leave them in place, whatever you want to do with the artifact you can do it in Bazel.

Eg. this rule should soon allow `bazel run :ngx-tabs-libdemo.publish` to publish it to npm
so you never care about the output tree!